pkg/prowgen/prowgen_config.go

```diff
@@ -257,6 +258,17 @@ func NewGenerateConfigs(ctx context.Context, r Repository, cc CommonConfig, opts
 				r.Org+"-"+r.Repo+"-"+branchName+"__"+variant+".yaml",
 			)
 
+			// Generate build config only for BinaryBuildCommand and TestBinaryBuildCommands appending to yaml root
+			// Used in client.yaml config
+			for _, customCfg := range r.CustomConfigs {
+				if len(customCfg.ReleaseBuildConfiguration.BinaryBuildCommands) > 0 {
+					cfg.BinaryBuildCommands = customCfg.ReleaseBuildConfiguration.BinaryBuildCommands
+				}
+				if len(customCfg.ReleaseBuildConfiguration.TestBinaryBuildCommands) > 0 {
+					cfg.TestBinaryBuildCommands = customCfg.ReleaseBuildConfiguration.TestBinaryBuildCommands
+				}
+			}
+
```

Comment:
This generate build commands based on the following yaml. It's custom config, but it needs to be in root of yaml doc.

```
- releaseBuildConfiguration:
          binary_build_commands: |
            make install
            make build-cross
          test_binary_build_commands: make test install
```

Reply:
Removed this bit, I'm testing out a new `Dockerfile` for artifacts image, that uses OpenShift CI pipeline src.

Reply:
https://github.com/openshift-knative/client/commit/3f329251741e3917e28e3b461d9c52d7afec9028#diff-6492e3702be91832f2bf8bb9063894f0849b3a8189b2b3622b41198fda31830a

Reply:
nice!